parts/k8s/kubernetesmasterresources.t

```diff
@@ -464,11 +483,17 @@
           "name": "nicLoopNode"
         },
         "dependsOn": [
+  {{if IsKubernetes}}
   {{if .MasterProfile.IsCustomVNET}}
           "[variables('nsgID')]"
   {{else}}
           "[variables('vnetID')]"
   {{end}}
+  {{else}}
+  {{if not .MasterProfile.IsCustomVNET}}
```

Comment:
are you missing `"[variables('nsgID')]"` here?

Reply:
yes, added